Packs/FeedMitreAttackv2/Playbooks/MITRE_-_Extract_and_Associate_to_Incident.yml

```diff
@@ -0,0 +1,241 @@
+    scriptarguments:
+      attack_pattern:
+        simple: ${MITREATTACK.value}
+    reputationcalc: 2
+    separatecontext: false
+    view: |-
+      {
+        "position": {
+          "x": 630,
+          "y": 590
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  '5':
+    id: '5'
+    taskid: 78c61601-dac1-4c2e-86f8-d868beab2f9b
+    type: title
+    task:
+      id: 78c61601-dac1-4c2e-86f8-d868beab2f9b
+      version: -1
+      name: Done
+      type: title
+      iscommand: false
+      brand: ''
+      description: ''
+    separatecontext: false
+    view: |-
+      {
+        "position": {
+          "x": 290,
+          "y": 760
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  '6':
+    id: '6'
+    taskid: f2c805ba-07f6-4550-8fdb-c03098941b75
+    type: condition
+    task:
+      id: f2c805ba-07f6-4550-8fdb-c03098941b75
+      version: -1
+      name: Are there techniques?
+      type: condition
+      iscommand: false
+      brand: ''
+      description: ''
+    nexttasks:
+      '#default#':
+      - '5'
+      yes:
+      - '3'
+    separatecontext: false
+    conditions:
+    - label: yes
+      condition:
+      - - operator: isNotEmpty
+          left:
+            value:
+              simple: MITREATTACK.value
+            iscontext: true
+    view: |-
+      {
+        "position": {
+          "x": 500,
+          "y": 380
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  '7':
+    id: '7'
+    taskid: c004b40d-2957-4ead-88f0-c63a111947bc
+    type: condition
+    task:
+      id: c004b40d-2957-4ead-88f0-c63a111947bc
+      version: -1
+      name: Is MITRE ATT&CK v2 available?
+      description: Returns 'yes' if integration brand is available. Otherwise returns 'no'
+      scriptName: IsIntegrationAvailable
+      type: condition
+      iscommand: false
+      brand: ''
+    nexttasks:
+      no:
+      - '5'
+      yes:
+      - '2'
+    scriptarguments:
+      brandname:
+        simple: MITRE ATT&CK v2
+    results:
+    - brandInstances
+    separatecontext: false
+    view: |-
+      {
+        "position": {
+          "x": 290,
+          "y": 30
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+view: |-
+  {
+    "linkLabelsPosition": {},
+    "paper": {
+      "dimensions": {
+        "height": 925,
+        "width": 720,
+        "x": 290,
+        "y": -100
+      }
+    }
+  }
+inputs:
+- key: TechniqueID
+  value: {}
+  required: false
```

Comment:
Are you sure that you want to leave that as a non-required?

Reply:
Yes works without an input as well
Preference to avoid mandatory inputs